### ISSUE TITLE
Hyper parameters for ChildTuning

### ISSUE THREAD

**Issue by LanJiang315** [alibaba/AliceMind#28]
Thanks a lot for all the details you provide in Appendix B for reproducibility! However, I still encounter some difficulties in reproducing the experiment. 
I noticed that you apply grid search. Could you please provide the specific <a href="https://www.codecogs.com/eqnedit.php?latex=p_F" target="_blank"><img src="https://latex.codecogs.com/gif.latex?p_F" title="p_F" /></a>, <a href="https://www.codecogs.com/eqnedit.php?latex=p_D" target="_blank"><img src="https://latex.codecogs.com/gif.latex?p_D" title="p_D" /></a> and learning rate for each task?

**Comment by RunxinXu** [comment]
Cola pf=pd=0.2
Rte pf=pd=0.3
Mrpc pf=0.4 pd=0.1
Stab pf=0.2 pd=0.2

It would be better to grid search the hyper-parameters.

**Comment by LanJiang315** [comment]
thx!!
